front/src/applications/stdcmV2/components/StdcmResults/StdcmResults.tsx

```diff
@@ -73,6 +80,104 @@ const StcdmResults = ({
     );
   }, [selectedSimulation]);
 
+  const infraId = useInfraID();
+  const pathProperties = usePathProperties(
```

Comment:
In `useStdcmResults` we are already calling `pathProperties`, so we can just add the pathProperties's props needed directly in `useStdcmResults` `['zones']` and return `zones` in `setPathProperties`

Reply:
agreed but all api calls to be refactored when the editoast part is merged :)

Reply:
I understand but if it's possible we can implement this feature the closest way to the final goal as we know the conflicts data will come from stdcm response. We don't have to keep everything for later 🙂

Reply:
Done :)

Reply:
Thanks :) 
I think we can add the logic when `response.status === 'conflicts'` directly in `useStdcm`. What do you think ?